_registries/hms-rc_registry.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: registry
+title:  "HMS Research Computing Container Registry"
+base: https://sregistry.hms.harvard.edu
+date:   2021-01-16 13:19:12
+author: vladdoster
+categories:
+- registry
+img: robots/hms_rc-robot.png

```

Comment:
Looks like there is one issue with showing your robot image, I'd suggest we try making this `hms-rc_robot.png`
```suggestion
img: robots/hms-rc_robot.png
```